src/main/tray/tray-menu-item/implementations/about-app-tray-item.injectable.ts

```diff
@@ -32,15 +32,10 @@ const aboutAppTrayItemInjectable = getInjectable({
       click: pipeline(
         async () => {
           await showApplicationWindow();
-
-          await showAbout();
+          showAbout();
```

Comment:
This looks like an avoidable orphan promise. Has `showAbout` changed to be sync instead of async?

Reply:
It never was `async`.